CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ project(
     HOMEPAGE_URL "https://github.com/ashvardanian/simsimd"
 )
 
-set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD 23)
```

Comment:
We shouldn't require the newest language standard support for the users.

Reply:
The new standard is required, https://github.com/ashvardanian/SimSIMD/issues/224.

Reply:
There should be another way to fix this without requiring C23. Let’s explore alternatives.